tests/qos/qos_helpers.py

```diff
@@ -35,6 +35,8 @@ def eos_to_linux_intf(eos_intf_name, hwsku=None):
     """
     if hwsku == "MLNX-OS":
         linux_intf_name = eos_intf_name.replace("ernet 1/", "sl1p").replace("/", "sp")
+    elif "Nokia" in hwsku:
```

Comment:
This check is not safe when hwsku is none.

Reply:
Sure. Will take care.